firebase-common/src/jsMain/kotlin/dev/gitlive/firebase/externals.kt

```diff
@@ -339,19 +339,14 @@ external object firebase {
             fun <T> runTransaction(func: (transaction: Transaction) -> Promise<T>): Promise<T>
             fun batch(): WriteBatch
             fun collection(collectionPath: String): CollectionReference
+            fun collectionGroup(collectionId: String): Query
             fun doc(documentPath: String): DocumentReference
             fun settings(settings: Json)
             fun enablePersistence(): Promise<Unit>
             fun clearPersistence(): Promise<Unit>
             fun useEmulator(host: String, port: Int)
         }
 
-        open class Timestamp {
-            val seconds: Double
-            val nanoseconds: Double
-            fun toMillis(): Double
```

Comment:
This toMiliis() removed, where it's used? Maybe we have to add it back to our Timestamp object?

Reply:
As far as I can see it is not used inside firebase-kotlin-sdk